jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -256,6 +256,10 @@ def mos_spec2d_parser(app, data_obj, data_labels=None, add_to_table=True,
         the mosviz table.
     data_labels : str, optional
         The label applied to the glue data component.
+    ext : int, optional
```

Comment:
The way HDUList is accessed, I think specs like `ext='SCI'` or `ext=('SCI', 1)` will also work though I guess we don't have to advertise it? 🤷